packages/vike-react-chakra/README.md

```diff
@@ -0,0 +1,97 @@
+<!-- WARNING: keep links absolute in this file so they work on NPM too -->
+
+[<img src="https://vike.dev/vike-readme.svg" align="right" height="90">](https://vike.dev)
+[![npm version](https://img.shields.io/npm/v/vike-react-chakra)](https://www.npmjs.com/package/vike-react-chakra)
+
+# `vike-react-chakra`
+
+[Installation](#installation)  
+[Basic usage](#basic-usage)  
+[`+chakra` Setting](#chakra-setting)  
+[See also](#see-also)  
+
+<br/>
+
+## Installation
+
+1. npm install @chakra-ui/react @emotion/react vike-react-chakra
+2. Extend `+config.js`:
+   ```js
+   // pages/+config.js
+
+   import vikeReact from 'vike-react/config'
+   import vikeReactChakra from 'vike-react-chakra/config'
+
+   export default {
+     // ...
+     extends: [vikeReact, vikeReactChakra]
+   }
+   ```
+
+> [!NOTE]
+> The `vike-react-chakra` extension requires [`vike-react`](https://vike.dev/vike-react).
+
+<br/>
+
+## Basic usage
```

Comment:
Maybe we can remove the whole `Basic usage` section with something like the following. WDYT?

> That's it, you can now use Chakra components for any of your component.
>
> ```jsx
> import { HStack, Button } from '@chakra-ui/react'
> const Demo = () => {
>   return (
>     <HStack>
>       <Button>Click me</Button>
>       <Button>Click me</Button>
>     </HStack>
>   )
> }
> ```

Reply:
So essentially removing the note and the heading.